data/json/monsters/zed-pupating.json

```
@@ -61,6 +61,145 @@
       ]
     }
   },
+  {
+    "id": "mon_brute_pupa_decoy",
+    "type": "MONSTER",
+    "name": { "str": "pupating brute" },
+    "description": "This muscular human corpse is wrapped in sticky black fibers that cover everything neck down.  Beneath the wrapping there is strange rhythmic movement grotesque to behold.",
```

```suggestion
    "description": "This muscular human corpse is wrapped in sticky black fibers that cover everything from the neck down.  Beneath the wrapping there is a strange rhythmic movement, grotesque to behold.",
```
How do the fibers look "sticky"?